### PR TITLE
Add roller importer

### PR DESCRIPTION
**Pull request by pmb00cs**
I've written an importer for apache roller, it is based on the wordpress importer but modified for the apache roller database structure. It appears to work for my apache roller blog.

### CODE REVIEW

**Comment by pmb00cs** [comment]
I've run the rubocop  job with auto correct, and also fixed the fact that it didn't import comments.

**Comment by ashmaroli** [comment]
@pmb00cs What this needs now is some tests.

**Comment by ashmaroli** [comment]
> what sort of tests do you want?

Awesome! :heart: your attitude!! :+1: 
At minimum, the test should cover `Importers::Roller.process(opts)`. That one method covers pretty much everything the Importer is supposed to do..

**Comment by pmb00cs** [comment]
I'll try and write some tests for that soon.

**Comment by ashmaroli** [comment]
@parkr Can we have at least one test covering the "query string" here..?
Honestly, I don't know how that string is ultimately used and how one can refactor it without any consequences..

File: lib/jekyll-import/importers/roller.rb [hunk 1]
```diff
@@ -97,28 +97,31 @@ def self.process(opts)
...
-             weblogentry.text         AS `content`,
-             weblogentry.summary      AS `excerpt`,
-             weblogentry.categoryid   AS `categoryid`,
-             roller_user.fullname     AS `author`,
-             roller_user.username     AS `author_login`,
-             roller_user.emailaddress AS `author_email`
-           FROM weblogentry AS `weblogentry`
-             LEFT JOIN roller_user AS `roller_user`
-               ON weblogentry.creator = roller_user.username"
+                 SELECT
```
[the remark this answers is not in the document]
Comment (pmb00cs) [reply lib/jekyll-import/importers/roller.rb]: You can ask, but I'm afraid the answer is probably not very satisfying.

This is the string I was having trouble being frozen, and in my naivety, I assumed the issue lay somewhere around here, so I made many changes to this string. I had thought I had put everything back as it was, but clearly not. After my changes and running `bundle exec rubocop --auto-correct` again this is what the result is. I'd love to blame rubocop, but it is far more likely my ad-hoc corrections were to blame.

Simply put it didn't, and this got included in my commit because of how I was editing the file. The fix is the two changes of `<<` to `+=` further down.

This string was changed with the addition of three lines to get the extra information I used to set the permalink.

[the remark this answers is not in the document]
Comment (ashmaroli) [reply lib/jekyll-import/importers/roller.rb]: Thank you for the response @pmb00cs 
I now see that you had added the following to lines to the "query string" as well, which was not immediately evident from the diff.
```
  LEFT JOIN weblog AS `weblog`
    ON weblogentry.websiteid = weblog.id
```

There is a bit of an inconsistency regarding the "query strings" in the class overall, though..
Here, the string starts with a `\n`..
```ruby
  posts_query = "
        SELECT
          weblogentry.id           AS `id`,
          ..."
```
elsewhere there is this:
```
  cquery =
    "SELECT
       weblogcategory.name AS `name`
```

Unfortunately, since I don't know how these strings are ultimately handled (no tests!), I won't be able to suggest a refactoring..

**Comment by ashmaroli** [comment]
@pmb00cs You can open another PR to add tests for this when ready.. I won't press you on adding tests right away..

**Comment by DirtyF** [comment]
@jekyllbot: merge +minor

**Comment by DirtyF** [comment]
@pmb00cs Let me know if it's OK to release a new minor version of this gem with current implementation. 💎

**Comment by pmb00cs** [comment]
So far I believe it does everything I needed of it. If you're happy with it I'd be very happy to have my contribution included in your project.

**Comment by DirtyF** [comment]
Shipped in latest release: https://github.com/jekyll/jekyll-import/releases/tag/v0.15.0 🎉 

:shipit:
